clients/samples/example_strmm.cpp

```diff
@@ -1,264 +0,0 @@
-/* ************************************************************************
- * Copyright (C) 2016-2023 Advanced Micro Devices, Inc. All rights reserved.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Why wasn't this just converted to show the new API use?

Reply:
Thanks for catching this. Added back example_strmm.cpp in next commit.